magento/magento-cloud-patches
# MAGECLOUD-4884: Update PageBuilder Patches for RCE

### Description
https://magento2.atlassian.net/browse/MAGECLOUD-4884

### Manual testing scenarios

1. Install 2.3.1 on cloud
2. Confirm patch `PB-319__fix_pagebuilder_module_2.3.1.patch` is applied
3. Install 2.3.2 on cloud
4. Confirm patch `PB-320__fix_pagebuilder_module__2.3.2.patch` is applied
5. Install 2.3.2-p1 on cloud
6. Confirm patch `PB-322__fix_pagebuilder_module__2.3.2-p1.patch` is applied
5. Install 2.3.2-p2 on cloud
6. Confirm patch `PB-322__fix_pagebuilder_module__2.3.2-p1.patch` is applied
7. Install 2.3.3 on cloud
8. Confirm patch `PB-323__fix_pagebuilder_module__2.3.3.patch` is applied
9. Confirm that the `MDVA-22979` patches are never applied

### Contribution checklist
